src/components/Toast/actions/styled.tsx

```diff
@@ -0,0 +1,30 @@
+import styled, { DefaultTheme } from 'styled-components';
+import { em } from 'polished';
+
+import { SIZES as WINDOW_SIZES } from '../../internal/useWindowSize';
+
+export const mdScreen = `min-width: ${em(WINDOW_SIZES.md)}`;
+
+export const SIZES = ['normal', 'reduced'] as const;
+export type Size = typeof SIZES[number];
+
+interface Props {
+  theme: DefaultTheme;
+  hasCloseButton: boolean;
+  size: Size;
+}
+
+export const toastSize = ({ theme, hasCloseButton, size }: Props) => {
+  const width = (() => {
+    console.log(size);
+    if (size === 'reduced') return 280;
+    return 380;
```

Comment:
I guess this magic number is from the design lol
If you so maybe we could make constants for them.

Reply:
Yeah. 😆  Changed!